lib/matplotlib/tri/_tri_wrapper.cpp

```diff
@@ -480,10 +480,7 @@ extern "C" {
 
 struct module_state
 {
-/* The Sun compiler can't handle empty structs */
```

Comment:
Is this comment referring to the `module_state` struct itself or the allocation that occurs at some other point? Because I don't see why this can't be remove entirely and just a straightforward `0` passed for the size.